### DISCUSSION TITLE
Feature request: Setting to turn off merge commit notifications

### DISCUSSION THREAD

**Discussion by cdeutsch** [community/community#32783] [Other Feature Feedback, Questions, & Ideas]
I'd like the ability to turn off "merge notifications" (second notification from the bottom)

They're very redundant (since they happen after every PR is merged)

![image](https://user-images.githubusercontent.com/404765/189682391-11c1660d-a263-4b88-99fb-a8dee524979d.png)

**Comment by cdeutsch** [comment]
Turns out these notifications are generated by Vercel.

If you scroll to the bottom of the changes you'll see a comment from Vercel Bot about "Successfully deployed to the following URLs" 🤦‍♂️

**Comment by spenserblack** [comment]
As you've noted, this is because of comments generated by Vercel.
If you want to turn these off, create a file called `vercel.json` with this:
```json
{
  "github": {
    "silent": true
  }
}
```
